tests/rustdoc-ui/invalid_associated_const.stderr

```diff
@@ -6,8 +6,9 @@ LL |     type A: S<C<X = 0i32> = 34>;
    |
 help: consider removing this associated item binding
    |
-LL |     type A: S<C<X = 0i32> = 34>;
-   |                ~~~~~~~~~~
+LL -     type A: S<C<X = 0i32> = 34>;
+LL +     type A: S<C = 34>;
+   |
```

Comment:
this should just be a removal span suggesting what to remove, there's no need to show the output after removing it